rpcd/playbooks/roles/elasticsearch/tasks/main.yml

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- include: upgrade_pre_flight.yml
+  when:
+    - logging_upgrade | bool
```

Comment:
Should this task file maybe use ``pre_flight_check`` to determine if ``upgrade_elasticsearch_pre`` should run? Maybe give it multiple when conditions? For example, add:

```
when:
  - pre_flight_check.rc == 0
# etc.
```